connectors/s3-tests/src/test/java/org/wso2/integration/ballerina/test/SampleTest.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.wso2.integration.ballerina.test;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Properties;
+
+import static io.restassured.RestAssured.given;
+
+public class SampleTest {
+    private static final String INPUTS_LOCATION = System.getenv("input_dir");
+    private static String externalip;
+    private static String nodeport;
+    private static String namespace;
+
+    static void test() throws Exception {
+        System.out.println();
+        InputStream input = new FileInputStream(INPUTS_LOCATION + "/deployment.properties");
+        Properties props = new Properties();
+        props.load(input);
+        externalip = props.getProperty("ExternalIP");
+        nodeport = props.getProperty("NodePort");
+        namespace = props.getProperty("namespace");
+        System.out.println(externalip);
+        System.out.println(nodeport);
+        System.out.println(namespace);
+        System.out.println("************INPUTS_LOCATION******************" + INPUTS_LOCATION);
+    }
+
+    @BeforeTest
+    public void init() throws Exception {
+        try {
+            test();
+            RestAssured.baseURI = "http://" + externalip + ":" + nodeport + "/amazons3/";
+
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Why `e.printStackTrace()`?

Reply:
There's no point in handling the exception because the test case will fail and the script executing the test case will exit on error. We're just printing the stacktrace just for debugging purpose.